Makefile

```diff
@@ -30,3 +30,13 @@ test-carthage:
 # Tests if current branch ships a valid Cocoapods project.
 test-cocoapods:
 		@cd dependency-manager-tests/cocoapods && $(MAKE)
+
+newversion:
+		@read -p "Enter version number: " version;  \
+		echo "// GENERATED FILE: Do not edit directly\n\ninternal let sdkVersion = \"$$version\"" > Sources/Datadog/Versioning.swift; \
+		sed "s/__DATADOG_VERSION__/$$version/g" DatadogSDK.podspec.meta > DatadogSDK.podspec; \
+		sed "s/__DATADOG_VERSION__/$$version/g" DatadogSDKObjc.podspec.meta > DatadogSDKObjc.podspec; \
```

Comment:
We use string replacement in [dependency-manager-tests](https://github.com/DataDog/dd-sdk-ios/tree/master/dependency-manager-tests), but the convention was to name meta files as `.src`. Could we do it here as well? 

* `DatadogSDK.podspec.src`
* `DatadogSDKObjc.podspec.src`

Reply:
Sure